qemu/tests/cfg/balloon_hotplug.cfg

```diff
@@ -13,6 +13,18 @@
     balloon_type_evict = evict
     balloon_type_enlarge = enlarge
     balloon_device = virtio-balloon-pci
+    Windows:
```

Comment:
As these cfg only used in variant "with_balloon_service", so I think it's better to move them into it.